numba/np/math/numbers.py

```diff
@@ -0,0 +1,1395 @@
+import math
+import numbers
+
+import numpy as np
+import operator
+
+from llvmlite import ir
+from llvmlite.ir import Constant
+
+from numba.core.imputils import (lower_builtin, lower_getattr,
+                                    lower_getattr_generic, lower_cast,
+                                    lower_constant, impl_ret_borrowed,
+                                    impl_ret_untracked)
```

Comment:
Same comment here. Should all registrations be commented out?

Reply:
Fixed in [93715b1](https://github.com/numba/numba/pull/9437/commits/93715b151e4350842fb227290eeda14d9f65c5ac)